src/main/daml/ContingentClaims/Core/Internal/Claim.daml

```diff
@@ -35,9 +35,9 @@ data Claim t x a o
       -- ^ The bearer acquires one unit of `a` *immediately*.
   | Give (Claim t x a o)
       -- ^ The obligations of the bearer and issuer are reversed.
-  | And with fst: Claim t x a o, snd: Claim t x a o, tail: [Claim t x a o]
+  | And with a1: Claim t x a o, a2: Claim t x a o, as: [Claim t x a o]
```

Comment:
After changing the from `fst : Claim` to `fst: (Text, Claim)` in the `Or` record I got a compiler error, hence had to find a different name for the record names. 

Once `Or` was changed (to `or1`, `or2`, `ors`, happy to hear other suggestions), it seemed reasonable to be consistent with the `And` record names.

Would you rather use `orFst` (`andFst`), ... ?

Reply:
> I got a compiler error,

Yes, that's expected because there are now two `fst` functions, with different return types (no function overloading in haskell)

> Would you rather use `orFst` (`andFst`), ... ?

No that's fine; I just took `fst`, `snd` from the Prelude, so users would recognise semantics.